js/index.js

```diff
@@ -1 +1,189 @@
-// JS goes here
\ No newline at end of file
+// JS goes here
+
+//Upon click, which the hamburger image to 'close' and change nav display: none => flex;
+
+let hamburger = document.querySelector('.nav-ham');
+let nav = document.querySelector('nav');
+let headNav = document.querySelector('.nav');
+
+hamburger.addEventListener('click', fromHam, {once: true});
+
+function fromHam(){
+    if(hamburger.src = '/img/nav-hamburger.png'){
+        hamburger.src = '/img/nav-hamburger-close.png';
+        nav.style.display = 'flex';
```

Comment:
might be preferable to handle this logic with adding and removing classes instead of styles